packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -312,10 +314,12 @@ export class EditableText extends AbstractComponent<IEditableTextProps, IEditabl
             // Chrome's input caret height misaligns text so the line-height must be larger than font-size.
             // The computed scrollHeight must also account for a larger inherited line-height from the parent.
             scrollHeight = Math.max(scrollHeight, getFontSize(this.valueElement) + 1, getLineHeight(parentElement));
-            // IE11 needs a small buffer so text does not shift prior to resizing
+            // IE11 & Edge needs a small buffer so text does not shift prior to resizing
+            scrollWidth = Browser.isEdge() ? scrollWidth + BUFFER_WIDTH_EDGE : scrollWidth;
+            scrollWidth = Browser.isInternetExplorer() ? scrollWidth + BUFFER_WIDTH_IE : scrollWidth;
```

Comment:
double ternaries are really awkward. please expand to `if ... else if ... else` (since both lines have the same `else` case)